blockchain.js

```diff
@@ -151,6 +151,15 @@ module.exports = class Blockchain {
     return this.instance;
   }
 
+  /**
+   * Check if Blockchain instance exists
+   * 
+   * @returns {Blockchain}
+   */
+  static isInstance() {
```

Comment:
I think "hasInstance" would be a better name for a static method.

Reply:
fixed